src/helpers/utility.js

```diff
@@ -0,0 +1,90 @@
+export const rollTheDice = () => {
```

Comment:
Hi again, I think you didn't see the roll the dice function You should check my useDice hook. This is also my bad because the function name rolls the dice but this function only rolls the dice for gpgp and others to the first location.

useDice() {
  const {dice, setDice} = useGameContext();

  function rollDice(diceDirection, diceStep) {
    let routes = [ "N", "NE", "E", "SE", "S", "SW", "W", "NW" ];
    const firstPlayerDirection = routes[diceDirection];
    const forbiddenDirections = [firstPlayerDirection, oppositeDirections[firstPlayerDirection]];

    for (const forbiddenDirection of forbiddenDirections ) {
      const index = routes.indexOf(forbiddenDirection);
      routes.splice(index, 1);
    }

    setDice({ ...dice, direction: diceDirection, step: diceStep, route: firstPlayerDirection  });
  }

  return { rollDice };
}